frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+export interface ScheduledEventsTableRowProps {
+  id: number;
+  name: string;
+  description: string;
+  enabled: boolean;
+  interval: number;
+  repeat: string;
+  interval_type: string;
+  start_date: string;
+}
+
+export const ScheduledEventsTableRow: React.SFC<ScheduledEventsTableRowProps> = (props) => {
```

Comment:
Prefer `FunctionComponent` instead of `SFC` ... this is deprecated. I'm sure you'll see this in some old components, but well, they're old ... we know better now :)